### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KateBlossom**
- [DEMO LINK](https://KateBlossom.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://KateBlossom.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -17,6 +29,85 @@
...
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+      <nav class="nav">
```
Comment (volodymyr-soltys97) [inline src/index.html:42]:
```suggestion
      </a>
      
      <nav class="nav">
```

File: src/index.html [hunk 2]
```diff
@@ -17,6 +29,85 @@
...
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```
Comment (volodymyr-soltys97) [inline src/index.html:52]: Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+nav a:hover {
+  text-decoration-color: #00acdc;
+  color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/style.css:65]: Use variable for this color

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,78 @@
...
+  color: #000;
+  line-height: 60px;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+nav a:hover {
```
Comment (volodymyr-soltys97) [inline src/style.css:63]: Need to use the classes selector

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,78 @@
...
+
+.is-active::after {
+  display: block;
+  content: '';
+  position: absolute;
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  background-color: #00acdc;
+  top: 56px;
```
Comment (volodymyr-soltys97) [inline src/style.css:76]:
```suggestion
  bottom: 0;
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's make your code better

**Review by l4st1m0za — APPROVED** [review]
Good job!

**Pull request closed without merging**